### ISSUE TITLE
Improve ability for first time contributors to get involved

### ISSUE THREAD

**Issue by dcwalk** [edgi-govdata-archiving/overview#179]
See conversation in #172 about standard labels. Being more accessible to new contributors has been brought up in slack a few times now, but winds up backburnered.

I think we should make a first attempt at this, proposing the following to mark complete:
- [ ] Add first-timer labels to active project repos and assign *at least* 1 issue
  * [x] [overview](https://github.com/edgi-govdata-archiving/overview)
  * [x] [edgi-scripts](https://github.com/edgi-govdata-archiving/edgi-scripts)
  * [x] [edgi-hubot-halpy](https://github.com/edgi-govdata-archiving/edgi-hubot-halpy)
  * [ ] [web-monitoring-processing](https://github.com/edgi-govdata-archiving/web-monitoring-processing)
  * [x] [eis-search](https://github.com/edgi-govdata-archiving/eis-search)
  * [ ] [web-monitoring-versionista-scraper](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper)
  * [ ] [web-monitoring-ops](https://github.com/edgi-govdata-archiving/web-monitoring-ops)
  * [x] [web-monitoring-db](https://github.com/edgi-govdata-archiving/web-monitoring-db)
  * [x] [web-monitoring](https://github.com/edgi-govdata-archiving/web-monitoring)
  * [x] [web-monitoring-ui](https://github.com/edgi-govdata-archiving/web-monitoring-ui)
  * [ ] [matterbridge-heroku](https://github.com/edgi-govdata-archiving/matterbridge-heroku)
  * [x] [edgi-website](https://github.com/edgi-govdata-archiving/edgi-website)
  * [ ] [archivers-greetingslack](https://github.com/edgi-govdata-archiving/archivers-greetingslack)
  * [ ] [halpy-ng](https://github.com/edgi-govdata-archiving/halpy-ng)
  * [ ] [100days](https://github.com/edgi-govdata-archiving/100days)
  * [x] [awesome-website-change-monitoring](https://github.com/edgi-govdata-archiving/awesome-website-change-monitoring)
  * [ ] [edgi-acronym-expander-browser-ext](https://github.com/edgi-govdata-archiving/edgi-acronym-expander-browser-ext)
  * [x] [video-call-landing-page](https://github.com/edgi-govdata-archiving/video-call-landing-page)
  * [ ] [eot-nomination-tool](https://github.com/edgi-govdata-archiving/eot-nomination-tool)
- [x] Verify all active project repos have contributing guidelines
  * [x] [overview](https://github.com/edgi-govdata-archiving/overview)
  * [x] [edgi-scripts](https://github.com/edgi-govdata-archiving/edgi-scripts)
  * [x] [edgi-hubot-halpy](https://github.com/edgi-govdata-archiving/edgi-hubot-halpy)
  * [x] [web-monitoring-processing](https://github.com/edgi-govdata-archiving/web-monitoring-processing)
  * [x] [eis-search](https://github.com/edgi-govdata-archiving/eis-search)
  * [x] [web-monitoring-versionista-scraper](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper)
  * [x] [web-monitoring-ops](https://github.com/edgi-govdata-archiving/web-monitoring-ops)
  * [x] [web-monitoring-db](https://github.com/edgi-govdata-archiving/web-monitoring-db)
  * [x] [web-monitoring](https://github.com/edgi-govdata-archiving/web-monitoring)
  * [x] [web-monitoring-ui](https://github.com/edgi-govdata-archiving/web-monitoring-ui)
  * [x] [matterbridge-heroku](https://github.com/edgi-govdata-archiving/matterbridge-heroku)
  * [x] [edgi-website](https://github.com/edgi-govdata-archiving/edgi-website)
  * [x] [archivers-greetingslack](https://github.com/edgi-govdata-archiving/archivers-greetingslack)
  * [x] [halpy-ng](https://github.com/edgi-govdata-archiving/halpy-ng)
  * [x] [100days](https://github.com/edgi-govdata-archiving/100days)
  * [x] [awesome-website-change-monitoring](https://github.com/edgi-govdata-archiving/awesome-website-change-monitoring)
  * [x] [edgi-acronym-expander-browser-ext](https://github.com/edgi-govdata-archiving/edgi-acronym-expander-browser-ext)
  * [x] [video-call-landing-page](https://github.com/edgi-govdata-archiving/video-call-landing-page)
  * [x] [eot-nomination-tool](https://github.com/edgi-govdata-archiving/eot-nomination-tool)
- [ ] Register our project as first-timer friendly on http://up-for-grabs.net

**Comment by dcwalk** [comment]
There are some other "newcomer friendly" OS project registries but I've lost 'em recently. I believe [OpenHatch](https://openhatch.org/search/) is no longer active. 

Anyone else have ones in mind?

**Comment by dcwalk** [comment]
And a link-dump:
- https://yourfirstpr.github.io/
- https://medium.com/@kentcdodds/first-timers-only-78281ea47455
- https://github.com/MunGell/awesome-for-beginners
- https://manishearth.github.io/blog/2016/01/03/making-your-open-source-project-newcomer-friendly/
- http://makeapullrequest.com/
- http://www.codenewbie.org/

**Comment by patcon** [comment]
Just to say it in the right place, I do lean toward `first-timer-only`, as I think it's more self-documenting of intention, but happy to kick that convo to a later time in the interest of not bikeshedding (because everything else is very actionable!)

**Comment by Frijol** [comment]
Per our new [stale issues policy](https://github.com/edgi-govdata-archiving/overview/blob/master/guidelines/project_guidelines.md#stale-issues-policy):

> This issue has been marked as stale because it has not had recent activity. It will be closed in seven days if no further activity occurs. If it should not be closed, please comment! Thank you for your contributions.

In the future, a robot will take care of this process!

**Comment by Frijol** [comment]
We should still do this! I might have some time this week

**Comment by Frijol** [comment]
[This spreadsheet](https://docs.google.com/spreadsheets/d/1IcPchZkOGjQnTUqRb9oH01Q9B7zZSAKtdU1F9mA51QA/edit?usp=sharing) notes which of EDGI's active repos have/don't have Contributing.md and issues marked for first-timers

**Comment by Frijol** [comment]
I added a checklist of active repos based on https://github.com/edgi-govdata-archiving/overview/issues/217

**Comment by Frijol** [comment]
Direct link to all EDGI "good-first-issue" issues: https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue+user%3Aedgi-govdata-archiving

**Comment by Frijol** [comment]
I checked over a lot of the repos that don't have "good first issue" labels and there really aren't a lot of issues that would be good for new folks–  ok to resolve?

I also looked over a bunch of the links for how to list our sites as up-for-grabs etc. but came away unsure how to actually list our stuff. I've done this before for other projects & remember it being easy (e.g. just enter a label name and repo URL) but I'm not seeing it now. Any suggestions?
